exporter/opentelemetry-exporter-datadog/tests/test_datadog_format.py

```diff
@@ -18,13 +18,11 @@
 from opentelemetry.exporter.datadog import constants, propagator
 from opentelemetry.sdk import trace
 from opentelemetry.trace import get_current_span, set_span_in_context
+from opentelemetry.trace.propagation.textmap import DictGetter
 
 FORMAT = propagator.DatadogFormat()
 
-
-def get_as_list(dict_object, key):
-    value = dict_object.get(key)
-    return [value] if value is not None else []
+getter = DictGetter()
```

Comment:
perhaps `carrier_getter` would be a better name for such a global var so it's more obvious when reading code way below this line.